File: core/src/main/java/com/github/nomad145/purepazaak/core/PurePazaak.java
package com.github.nomad145.purepazaak.core.core;

import playn.core.Image;
import playn.core.Platform;
import playn.scene.ImageLayer;
import playn.scene.SceneGame;

public class PurePazaak extends SceneGame {

  public PurePazaak (Platform plat) {
    super(plat, 33); // update our "simulation" 33ms (30 times per second)

    // create and add background image layer
    Image bgImage = plat.assets().getImage("images/bg.png");
    ImageLayer bgLayer = new ImageLayer(bgImage);
    // scale the background to fill the screen
    bgLayer.setSize(plat.graphics().viewSize);
    rootLayer.add(bgLayer);
  }
}

File: core/src/main/kotlin/com/github/nomad145/purepazaak/core/PurePazaak.kt
package com.github.nomad145.purepazaak.core;

import playn.core.Image;
import playn.core.Platform;
import playn.scene.ImageLayer;
import playn.scene.SceneGame;

class PurePazaak(var platform: Platform, var frameRate: Int) : SceneGame(platform, frameRate) {
  init {
    setBackground()
  }

  private fun setBackground() {
    val backgroundImage = platform.assets().getImage("images/background.png")
    val backgroundLayer = ImageLayer(backgroundImage)

    backgroundLayer.setSize(platform.graphics().viewSize)

    rootLayer.add(backgroundLayer)
  }
}
